helm/kubernetes-metrics-server-chart/templates/np.yaml

```diff
@@ -0,0 +1,27 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  namespace: {{ .Values.namespace }}
+  name: {{ .Values.name }}
+  labels:
+    app: {{ .Values.name }}
+    giantswarm.io/service-type: "managed"
+spec:
+  podSelector:
+    matchLabels:
+      app: {{ .Values.name }}
+  ingress:
+  - ports:
+    - port: {{ .Values.port }}
+      protocol: TCP
+  egress:
+  - to:
+    - ipBlock:
+        cidr: 10.0.0.0/8
```

Comment:
Same question as I did for ksm, do we need to change for node IP ranges which are not the default ones?

Reply:
Same answer - this allows all the private ips. 

So, those services which needs access to some external IPs has egress policy which allows all egress access (as there is no way to setup dns based restriction)

And those which do not need externall access - have egress policy only for accessing internal ips (that covers any host network ip within control-plane and tenant )